vllm_ascend/distributed/device_communicators/pyhccl.py

```diff
@@ -0,0 +1,168 @@
+#
```

Comment:
please move `vllm_ascend/communicator.py` to `vllm_ascend/distributed` module as well.